packages/shared/src/components/LoginButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import { Button } from './buttons/Button';
+import AuthContext from '../contexts/AuthContext';
+import FeaturesContext from '../contexts/FeaturesContext';
+import { getFeatureValue } from '../lib/featureManagement';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../hooks/analytics/useAnalyticsQueue';
+
+const getAnalyticsEvent = (
+  eventName: string,
+  copy: string,
+): Partial<AnalyticsEvent> => ({
+  event_name: eventName,
+  target_type: 'signup button',
+  target_id: 'header',
+  feed_item_title: copy,
+});
+
+export default function LoginButton(): ReactElement {
+  const { showLogin } = useContext(AuthContext);
+  const { flags } = useContext(FeaturesContext);
+  const { trackEvent } = useContext(AnalyticsContext);
+  const copy = getFeatureValue('signup_button_copy', flags) || 'Login';
```

Comment:
Only suggestion is to change this to buttonCopy to not give the impression it copies anything.

Reply:
Sure! I fixing now